truss-chains/truss_chains/definitions.py

```diff
@@ -249,30 +277,24 @@ def get_baseten_api_key(self) -> str:
 class TrussMetadata(generics.GenericModel, Generic[UserConfigT]):
     """Plugin for the truss config (in config["model_metadata"]["chains_metadata"])."""
 
-    user_config: UserConfigT = pydantic.Field(default=None)
+    user_config: UserConfigT
     chainlet_to_service: Mapping[str, ServiceDescriptor] = {}
 
 
-class ABCChainlet(Generic[UserConfigT], abc.ABC):
-    remote_config: ClassVar[RemoteConfig]
+class ABCChainlet(abc.ABC):
+    remote_config: ClassVar[RemoteConfig] = RemoteConfig(docker_image=DockerImage())
```

Comment:
just a note: I think even if we do set a default here, our examples should have this filled out, so it's quicker to know how to modify

Reply:
One other default we could consider is having RemoteConfig() defaulting the `docker_image`

Reply:
Agree, let's keep it in examples as a pointer.

The original idea for docker image as mandatory was to "force" users to be intentional, but not that the whole thing is optional anyway, we can also drop that requirement...